packages/sites/src/sites.ts

```diff
@@ -2,16 +2,15 @@
  * Apache-2.0 */
 
 import { request, IRequestOptions } from "@esri/arcgis-rest-request";
-import { IItem } from "@esri/arcgis-rest-common-types";
-import { getItem, getItemData } from "@esri/arcgis-rest-items";
+import { getHubUrl } from "@esri/hub-domains";
 
 /**
  * Fetch the domains associated with a Hub Site.
  * @param siteId - Identifier of the Hub Site
  * @param requestOptions - request options that may include authentication
  * @returns A Promise that will resolve with the domains associated with the site.
  */
-export function fetchDomains(
+export function fetchSiteDomains(
```

Comment:
this is a breaking name change.

Reply:
Not sure why I changed this... reverted...